lib/post_destroyer.rb

```diff
@@ -291,6 +292,11 @@ def trash_public_post_actions
     end
   end
 
+  def trash_revisions
+    return unless permanent?
+    @post.revisions.map(&:destroy!)
```

Comment:
```suggestion
    @post.revisions.each(&:destroy!)
```

We're not using the resulting `true`-filled array 😃